frontend/javascripts/admin/organization/upgrade_plan_modal.tsx

```diff
@@ -308,16 +309,17 @@ const CREDIT_COST_USD = 5.75;
 
 function OrderWebknossosCreditsModal({ destroy }: { destroy: () => void }) {
   const userInputRef = useRef<HTMLInputElement | null>(null);
-  const [costsAsString, setCostsAsString] = useState<string>("5€/5.75$");
-  const recalculateCosts = (newCosts: number | null) => {
+  const [creditCostAsString, setCreditCostsAsString] = useState<string>("5€/5.75$");
+  const [creditAmount, setCreditAmount] = useState<number | null>(1);
+  useEffect(() => {
     // TODOM: Remove magic numbers. E.g. put them into application configuration.
```

Comment:
todo

Reply:
ok should be done now

Reply:
I still see the "TODOM" comment in the pr. maybe something is weird with my diff view, though?

Reply:
Oh right, I forgot to delete this comment. Thanks for mentioning :+1: